### ISSUE TITLE
Split chapters out into seperate asciidoc files

### ISSUE THREAD

**Issue by PardusEidolon** [PardusEidolon/Halo-Editing-Kit-Cookbook#9]
# Adoc Cross References

Instead of the main adoc file having information on all the tools and guides for the H3EK, I was thinking of taking chapters out of the main file and putting them into separate adoc files, which then can be referenced in the main adoc file.

For example, the guides should be their adoc files respectively, referenced in the main adoc file, to reduce clogging. Ideally, the main file should reference a collection of many adoc files that isolate information making it easier for the viewer to read.

### Cross-Reference Tasks
- [ ] Figure out how to navigate between source-files
- [ ] Separate chapters from the main adoc file into their adoc files
- [ ] Reference new adoc files in the main adoc files

### Addtitional Information
- [AsciiDoc Syntax Quick Reference](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/)
- [Document to Document Cross References](https://docs.asciidoctor.org/asciidoc/latest/macros/inter-document-xref/)

**Comment by A267-dev** [comment]
I Agree that this would be the best way to approach this, and makes for cleaner source-control
